packages/api-wcp/src/createWcp.ts

```diff
@@ -50,25 +50,22 @@ export const createWcp = async (): Promise<WcpContextObject> => {
             return;
         }
 
+        let jsonParseError, json;
+
         try {
-            const json = await response.json();
-            console.error(
-                `An error occurred while trying to ${operation} user seats.`,
-                response.status,
-                response.statusText,
-                response.json
-            );
-            throw new Error(json.message);
+            json = await response.json();
         } catch (e) {
-            console.error(
-                `An error occurred while trying to ${operation} user seats.`,
-                response.status,
-                response.statusText
-            );
-            throw new Error(
-                `An error occurred while trying to ${operation} user seats. Please check logs for more info.`
-            );
+            jsonParseError = e;
+        }
+
+        let message = `An error occurred while trying to ${operation} user seats.`;
```

Comment:
Maybe let's try this form of message:
Instead of `An error occurred while trying to`, we can say `Failed to`, which essentially means the same thing, but is 5 times shorter. It's better from the user perspective, and us typing. You just get the message across faster.

I'd make it `Failed to ${operation} user seats.`. And if you agree, I'd generally make this a standard way we write these messages.